### DISCUSSION TITLE
ib_insync group on io

### DISCUSSION THREAD

**Discussion by Sdoof** [mattsta/ib_insync#3] [General]
Hi there, How can we recover all the discussions on the IO group, this was a goldmine for anyone keen to learn the finess of the ib_insync package, it seems like we woulkd all benefit from trying to unarchive it. Any ideas of what that would take?

**Comment by johnjohnk2** [comment]
Hi @Sdoof I have them in a .zip compressed archive. Just ask it to me on discord
